medusa/server/api/v2/config.py

```diff
@@ -328,3 +396,14 @@ def data_statuses():
         section_data['strings'] = common.statusStrings
 
         return section_data
+
+    @staticmethod
+    def data_metadata():
+        """Metadata."""
+        section_data = NonEmptyDict()
+
+        section_data['metadataProviders'] = {
+            provider.to_json().get('id'): provider.to_json() for provider in app.metadata_provider_dict.values()
+        }
```

Comment:
- This results in `metadata.metadataProviders`.
- No reason to call `provider.to_json()` twice.
- Don't use `.get('id')` because it could result in a `None` key, I prefer it would raise KeyError if there's no `id`.
- Use `itervalues` from `six` please.
```py
section_data['providers'] = NonEmptyDict()
for provider in itervalues(app.metadata_provider_dict):
    json_repr = provider.to_json()
    section_data['providers'][json_repr['id']] = json_repr
```